libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -582,9 +582,14 @@ bool AP_InertialSensor::calibrate_accel(AP_InertialSensor_UserInteract* interact
           We use the original board rotation for this sample
         */
         Vector3f level_sample = samples[0][0];
+        level_sample.x *= new_scaling[0].x;
+        level_sample.y *= new_scaling[0].y;
+        level_sample.z *= new_scaling[0].z;
+        level_sample += new_offsets[0];
         level_sample.rotate(saved_orientation);
 
-        _calculate_trim(level_sample, trim_roll, trim_pitch);
+        trim_pitch = atan2(level_sample.x,pythagorous2(level_sample.y,level_sample.z));
```

Comment:
You guys are killing me! Please use atan2f() for floats. Now you made me re-scan the whole repo for this and I found a dozen more lingering around in various places. Please be careful.